### DISCUSSION TITLE
Retrieval scores for similarity search

### DISCUSSION THREAD

**Discussion by heiko-braun** [lancedb/lancedb#823] [Q&A]
Hi, I am using Lancedb in a RAG scenario and wonder if it's possible to retrieve the scores for each document that results from the similarity search?

An example would be great, or a pointer to the right documentation.

Thanks a lot!

**Comment by eddyxu** [comment]
Hi, @heiko-braun 

Sorry for late reply. Each vector query will return  a `_distance` column, which is the score of the vector search. Is it what you are looking for?

Btw, we have a discord channel, posting there could get much timely answers. 

https://discord.gg/zMM32dvNtd

Best,
